cg/meta/workflow/analysis.py

```diff
@@ -312,6 +312,62 @@ def link_fastq_files_for_sample(
             self.fastq_handler.concatenate(linked_reads_paths[read], concatenated_paths[read])
             self.fastq_handler.remove_files(value)
 
+    def get_metadata_for_nanopore_sample(self, sample_obj: models.Sample) -> List[dict]:
```

Comment:
Similar to the fastqhandlers, both these functions can move to mutant.py under `MutantAnalysisAPI`.

Reply:
moved and it works